xocto/fields/ranges.py

```diff
@@ -0,0 +1,68 @@
+"""
+Adapter fields for postgres range types that return xocto.Range python objects.
+"""
+import datetime
+import json
+from typing import Optional
+
+from django.contrib.postgres import fields as pg_fields
+from django.contrib.postgres.fields import ranges as pg_ranges
+from django.contrib.postgres.fields import utils as pg_utils
+from django.db import models
+
+from xocto import ranges
+
+
+class FiniteDateRangeField(pg_fields.DateRangeField):
+    """
+    A DateRangeField with Inclusive-Inclusive [] bounds.
+
+    The underlying postgres type will always store the range as [). This field
```

Comment:
I wonder if this comment should expand on the notation you've used here

Reply:
I'm not keen to be repeating Word-Word [symbol-symbol) everywhere. I'm thinking of defaulting to just symbols as that's common-enough symbolism. Thoughts?

Reply:
Perhaps a full explanation somewhere and a reference to that where you use them?